src/components/KYCWall/kycWallPropTypes.js

```diff
@@ -25,6 +27,15 @@ const propTypes = {
 
     /** When the button is opened via an IOU, ID for the chatReport that the IOU is linked to */
     chatReportID: PropTypes.string,
+
+    /** List of cards */
+    cardList: PropTypes.objectOf(cardPropTypes),
+
+    /** List of bank accounts */
+    bankAccountList: PropTypes.objectOf(bankAccountPropTypes),
+
+    /** Are we loading payment methods? */
+    isLoadingPaymentMethods: PropTypes.bool,
```

Comment:
This prop is removed in this PR https://github.com/Expensify/App/pull/12299 and has not been used in the `BaseKYCWall`.